### DISCUSSION TITLE
Grable for arm32

### DISCUSSION THREAD

**Discussion by ExtinctDrift19** [termux/termux-packages#11236] [Q&A] [answered]
bit 32 grable

**Comment by 2096779623** [comment, marked as answer]
`pkg i gradle`.

**Reply by sylirre** [reply]
Yeah, it is `gradle` actually and it is architecture-independent (available for all devices).

**Comment by ExtinctDrift19** [comment]
![Screenshot_20220713-181432_Termux](https://user-images.githubusercontent.com/82643977/178769100-11dbd75d-31a0-4415-8bbe-2a0f0d1dfa80.jpg)

**Reply by licy183** [reply]
It names `gradle` rather than `grable` I think. You should correct the typo, or open a package request if there is some package naming `grable`...
